src/test/java/com/github/javafaker/InternetTest.java

```diff
@@ -276,4 +277,16 @@ public void testFarsiIDNs() {
         assertThat(f.internet().safeEmailAddress(), not(isEmptyOrNullString()));
         assertThat(f.internet().url(), not(isEmptyOrNullString()));
     }
+
+    @Test
+    public void testUserAgent() {
+        Internet.USER_AGENT[] agents = Internet.USER_AGENT.values();
+        for(Internet.USER_AGENT agent : agents){
+            assertThat(faker.internet().userAgent(agent), IsNot.not(isEmptyOrNullString()));
```

Comment:
There's the static import already for IsNot so you shouldn't need to fully qualify it